com.unity.perception/Runtime/Randomization/Scenarios/PerceptionScenario.cs

```diff
@@ -20,29 +20,13 @@ public abstract class PerceptionScenario<T> : Scenario<T> where T : ScenarioCons
         /// </summary>
         MetricDefinition m_IterationMetricDefinition;
 
-        /// <summary>
-        /// The scriptable render pipeline hook used to capture perception data skips the first frame of the simulation
-        /// when running locally, so this flag is used to track whether the first frame has been skipped yet.
-        /// </summary>
-        protected bool m_SkippedFirstFrame;
-
-        /// <inheritdoc/>
-        protected override bool isScenarioReadyToStart
-        {
-            get
-            {
-                if (!m_SkippedFirstFrame)
-                {
-                    m_SkippedFirstFrame = true;
-                    return false;
-                }
-                return true;
-            }
-        }
+        protected override bool isScenarioReadyToStart => Time.frameCount >= 2;
 
         /// <inheritdoc/>
         protected override void OnAwake()
         {
+            foreach (var perceptionCamera in FindObjectsOfType<PerceptionCamera>())
```

Comment:
This works for any apps using Scenario, but this should also be addressed when not using a Scenario.

Reply:
I think the underlying issue was that scenarios were not starting at the same time the perception camera starts capturing. If a user is not using a scenario, then the issue is no longer valid. I'm not sure I understand your feedback in this case.

Reply:
This code is disabling the perception camera. If it's the scenario timing that needs adjusting, we shouldn't need to touch the camera